public/de/index.md

```diff
@@ -85,7 +85,7 @@ classes:
 </blockquote></div>
 }!}
 
-Seit es die menschliche Zivilisation gibt, gibt es Menschen, deren körperliche Erfahrungen nicht mit denen ihres inneren Geschlechts und den physischen Merkmalen ihres Körpers übereinstimmten. Die Gala, eine Priesterklasse mittleren Geschlechts des sumerischen Reiches, traten vor über 4500 Jahren auf. Die indigenen Kulturen Nordamerikas haben weit vor dem europäischen Kolonialismus [ein drittes Geschlecht anerkannt](https://de.wikipedia.org/wiki/Drittes_Geschlecht) und tun dies bis heute. Der römische Kaiser Elagabalus (218 n. Chr.) Bestand darauf, als Lady statt als Lord bezeichnet zu werden, und stellte sogar eine Belohnung bereit für jeden, der eine Genitalrekonstruktion durchführen konnte. 
+Seit es die menschliche Zivilisation gibt, gibt es Menschen, deren körperliche Erfahrungen nicht mit denen ihres inneren Geschlechts und den physischen Merkmalen ihres Körpers übereinstimmten. Die [Gala](https://de.abcdef.wiki/wiki/Gala_(priests)), eine Priesterklasse mittleren Geschlechts des sumerischen Reiches, traten vor über 4500 Jahren auf. Die indigenen Kulturen Nordamerikas haben weit vor dem europäischen Kolonialismus [ein drittes Geschlecht anerkannt](https://de.wikipedia.org/wiki/Drittes_Geschlecht) und tun dies bis heute. Der römische Kaiser Elagabalus (218 n. Chr.) Bestand darauf, als Lady statt als Lord bezeichnet zu werden, und stellte sogar eine Belohnung bereit für jeden, der eine Genitalrekonstruktion durchführen konnte. 
```

Comment:
Is abcdef.wiki a legit webpage?

Reply:
Yeah, sorry for that one. I was just searching for some reference for this one, as I missed some more sources on the first page regarding trans history. Haven't found much, so I used this one. Remove it, if you don't like it :)
The most important changes are the typos or other grammar issues.